crmsh/bootstrap.py

```diff
@@ -1777,6 +1821,29 @@ def join_ssh(seed_host):
         error("Can't invoke crm cluster init -i {} ssh_remote on {}".format(_context.default_nic_list[0], seed_host))
 
 
+def fetch_public_key_from_remote_node(node):
+    """
+    Fetch public key file from remote node
+    Return a temp file contains public key
+    Return None if no key exist
+    """
+
+    # For dsa, might need to add PubkeyAcceptedKeyTypes=+ssh-dss to config file, see
+    # https://superuser.com/questions/1016989/ssh-dsa-keys-no-longer-work-for-password-less-authentication
+    for key in ("id_rsa", "id_ecdsa", "id_ed25519", "id_dsa"):
+        public_key_file = "/root/.ssh/{}.pub".format(key)
+        cmd = "ssh -oStrictHostKeyChecking=no root@{} 'test -f {}'".format(node, public_key_file)
```

Comment:
@arbulu89 , This will be called under these scenarios:

- On join node, fetch from init node, just after append join node's public key to init node's authorized_keys, should be passwordless now
- On init node, `ssh_passwordless_remote` stage, fetch from join node, this step was after `join_ssh` which has already setup passwordless between init and join node
- On init node, `ssh_passwordless_remote` stage, fetch from other cluster nodes, assuming that other cluster nodes already configured passwordless between the init node

On above scenarios we all assuming that it's already **should** passwordless configured before running `test -f` remotely.
But yes, you were right, there should also have some `corner case`.

How about here we also use `utils.check_ssh_passwd_need` to detect the connection to this remote node, if just in case there was not passwordless, here we prompted a meaningful message hints we will ask the password multi times.

What do you think about it?